capstone-frontend/src/components/BookPage.jsx

```diff
@@ -79,7 +79,7 @@ export class BookPage extends Component {
                         starSpacing='10px'
                         starRatedColor='gold'
                       />
-                      {book.avgRating === 0 && <p id='rating-label'>No Ratings</p>}
+                      {book.avgRating < 1 && <p id='rating-label'>No Ratings</p>}
```

Comment:
What was the reasoning for changing this?

Reply:
The original goal was for `No Ratings` to pop up next to the stars when there were no ratings found on the backend. On the Google Books site, there is no way to give a book a rating less than 1. On the backend I set the `avgRating` variable to be -1 in the case of no ratings.

With the equality check here for 0, it does not do what I originally intended for it to do, this string did not appear as it should've when there were no ratings.